### DISCUSSION TITLE
Density Map

### DISCUSSION THREAD

**Discussion by KBattousaiH** [OpenDrift/opendrift#1006] [Q&A]
Hi!
 Has anyone made a density map with the "o.write_netcdf_density_map()" option? I used it like 
 
 "o.write_netcdf_density_map('density.nc', pixelsize_m=300)"
 
and it gave me the following:

Traceback (most recent call last):
  File "/data2/matlab/MOSA_BGQ/mosa_drift_back.py", line 39, in <module>
    o.write_netcdf_density_map('density.nc', pixelsize_m=300)
  File "/home/matlab/opendrift/opendrift/models/basemodel.py", line 4762, in write_netcdf_density_map
    nc = Dataset(filename, 'w')
NameError: name 'Dataset' is not defined

How can I solve it? Is there another way to make density maps?

**Comment by knutfrode** [comment]
Hi,

This method works for me, and also the line numbers are quite different
from your error message.
Can you please try to update your OpenDrift to the newest version?
Remember then also to update your conda environment:
https://opendrift.github.io/install.html

Best regards from Knut-Frode


tor. 27. okt. 2022 kl. 23:53 skrev Catalina Peñailillo <
***@***.***>:
